test/test.mk

```diff
@@ -38,7 +38,8 @@ test-sh:
 	$(test-dir)/env $(base) $(mode) $(binaries) -i /bin/sh
 
 test-spec:
-	@sed 's/{REPLACE}/$(subst ",,$(INFIX_NAME)) $(GIT_VERSION)/'  $(spec-dir)/Readme.adoc.in > $(spec-dir)/Readme.adoc
+	@esc_infix_name="$(echo $(INFIX_NAME) | sed 's/\//\\\//g')"; \
```

Comment:
Why?

Reply:
When INFIX_NAME contains '/' character, the `sed` command gets confused when '/' is used as a delimiter